opm/parser/eclipse/EclipseState/Schedule/ScheduleEnums.hpp

```diff
@@ -219,6 +219,23 @@ namespace Opm {
 
         const std::string ControlEnum2String( GroupProduction::ControlEnum enumValue );
         GroupProduction::ControlEnum ControlEnumFromString( const std::string& stringValue );
+
+        enum GuideRateDef {
```

Comment:
Use the "new" `class enum` - that is a *slight* improvement over the old style enums in that the symbols are accessed with class prefix in the code:

#### Old enum style - as in current code:
```C++
GuideRateDef guide_rate;
...
if (guide_rate == OIL)
   ...
```

#### New enum style
```C++
class enum GuideRateDef {
      OIL = 0,
      WAT = 1,
      GAS = 2, 
      ....
}

GuideRateDef guide_rate;
...
if (guide_rate == GuideRateDef::OIL) 
    ....
```